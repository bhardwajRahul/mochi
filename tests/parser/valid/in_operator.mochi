let x = "w" in "hello"
