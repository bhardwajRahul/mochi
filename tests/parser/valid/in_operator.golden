(program
  (let x
    (binary in (string w) (string hello))
  )
)
